### PR TITLE
Swords icons and some material properties

### PR DESCRIPTION
**Pull request by comma**
Swords now are dull if made from non hard materials.
Need HARD for edge, RIGID for at least poking quality.
Split icons, replaced icons with ones made by Deer and my variations of those.

Lots of changes are repathse to get all sords under same path.
image
image

### CODE REVIEW

File: code/game/objects/items/weapons/material/swords.dm [hunk 1]
```diff
@@ -15,28 +15,60 @@
...
+/obj/item/material/sword/experimental_mob_overlay(mob/user_mob, slot)
+	var/image/res = ..()
+	//Do not color scabbarded blades
+	if(applies_material_colour && (slot == slot_back_str || slot == slot_belt_str))
+		res.color = null
+	return res
+
+/obj/item/material/sword/wood
+	material = MAT_WOOD
+	draw_handle = FALSE
```
Comment (Techhead0) [inline code/game/objects/items/weapons/material/swords.dm:48]: Do you want to also make player-made woodcrafts set this?

File: code/_helpers/unsorted.dm [hunk 2]
```diff
@@ -859,4 +859,14 @@ var/list/WALLITEMS = list(
...
+
+/proc/get_mutable_overlay(icon, icon_state, color, flags = RESET_COLOR | RESET_ALPHA, plane = FLOAT_PLANE)
+	var/mutable_appearance/res = new()
+	res.icon = icon
+	res.icon_state = icon_state
+	res.color = color
+	res.appearance_flags = flags
+	res.plane = plane
+	res.layer = FLOAT_LAYER
+	return res
```
Comment (Techhead0) [inline code/_helpers/unsorted.dm:872]: Probably could go in `code/_helpers/icons.dm`

Comment (comma) [reply code/_helpers/unsorted.dm:872]: I thought about that but it's a scary dark place where super arcane icon ops shit lives

File: code/game/objects/items/weapons/material/swords.dm (hunk 1 above)
Comment (comma) [reply code/game/objects/items/weapons/material/swords.dm:48]: currently sords arent' craftable, so will think of some way to set handles when doing that.

File: code/game/objects/items/weapons/material/swords.dm [hunk 3]
```diff
@@ -15,28 +15,60 @@
...
+
+/obj/item/material/sword/update_force()
+	if(material?.hardness < MAT_VALUE_HARD)
+		edge = 0
+		attack_verb = list("attacked", "stabbed", "jabbed", "smacked", "prodded")
+		hitsound = 'sound/weapons/pierce.ogg'
+	if(material?.hardness < MAT_VALUE_RIGID)
+		sharp = 0
+		attack_verb = list("attacked", "smashed", "jabbed", "smacked", "prodded", "bonked")
+		hitsound = "chop"
```
Comment (Techhead0) [inline code/game/objects/items/weapons/material/swords.dm:28]: Not a sound file

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**